src/base/solvers/configEnvVars.m

```diff
@@ -186,7 +186,8 @@ function configEnvVars(printLevel)
 
         % check for 32-bit
         if ~isempty(strfind(computer('arch'), '64'))
-            subDir = [filesep, osPath, filesep, 'matlab'];
+            subDir = [filesep, 'matlab'];
+            %subDir = [filesep, osPath, filesep, 'matlab'];
         end
```

Comment:
Thanks for this detailed reporting. I understand that there may be a header missing in `configEnvVars.m`.

Lines 13-64 of `configEnvVars.m` are initialisations only, and the method that is used is the following (current comments in the code):

1. try retrieving the solver path from the environment variables
2. loop through the list of possible directories
3. check if the solver is already on the MATLAB path

Which step do you recommend to be the first line approach?

As for the installation instructions and as we agreed, we will add a link to the official installation instructions whenever possible. Now, there are however some additional instructions that are required in order to make the standard solver installation work with The COBRA Toolbox. For instance, did you set:
```sh
export GUROBI_PATH="${GUROBI_HOME}"
```
This is marked as an extra step in the installation instructions, and is required as `/usr/local/bin/gurobi702` is not the default path.